x-pack/plugins/data_visualizer/public/application/data_drift/constants.ts

```diff
@@ -13,39 +13,31 @@ export const DATA_COMPARISON_TYPE = {
   UNSUPPORTED: 'unsupported',
 } as const;
 
-export const NUMERIC_TYPE_LABEL = i18n.translate(
-  'xpack.dataVisualizer.dataComparison.numericLabel',
-  {
-    defaultMessage: 'Numeric',
-  }
-);
+export const NUMERIC_TYPE_LABEL = i18n.translate('xpack.dataVisualizer.dataDrift.numericLabel', {
+  defaultMessage: 'Numeric',
+});
 export const CATEGORICAL_TYPE_LABEL = i18n.translate(
-  'xpack.dataVisualizer.dataComparison.categoricalLabel',
+  'xpack.dataVisualizer.dataDrift.categoricalLabel',
   {
     defaultMessage: 'Categorical',
   }
 );
 
-export const UNSUPPORTED_LABEL = i18n.translate(
-  'xpack.dataVisualizer.dataComparison.UnsupportedLabel',
-  {
-    defaultMessage: 'Unsupported',
-  }
-);
+export const UNSUPPORTED_LABEL = i18n.translate('xpack.dataVisualizer.dataDrift.UnsupportedLabel', {
+  defaultMessage: 'Unsupported',
+});
 
-export const REFERENCE_LABEL = i18n.translate(
-  'xpack.dataVisualizer.dataComparison.referenceLabel',
-  {
-    defaultMessage: 'Reference',
-  }
-);
+export const REFERENCE_LABEL = i18n.translate('xpack.dataVisualizer.dataDrift.referenceLabel', {
+  defaultMessage: 'Reference',
+});
 
-export const COMPARISON_LABEL = i18n.translate(
-  'xpack.dataVisualizer.dataComparison.productionLabel',
-  {
-    defaultMessage: 'Comparison',
-  }
-);
+export const COMPARISON_LABEL = i18n.translate('xpack.dataVisualizer.dataDrift.comparisonLabel', {
+  defaultMessage: 'Comparison',
+});
+
+export const PRODUCTION_LABEL = i18n.translate('xpack.dataVisualizer.dataDrift.productionLabel', {
```

Comment:
Is this label needed, or are we adding an extra term unnecessarily as everywhere else in the UI it looks like we are using the term 'comparison'? The only place I can find where we use it in the UI is on the bottom histogram chart.

<img width="304" alt="image" src="https://github.com/elastic/kibana/assets/7405507/c5ce439c-41cd-4b80-a71d-6d020695299e">

Reply:
Updated labeling here [`f2f5a38` (#162853)](https://github.com/elastic/kibana/pull/162853/commits/f2f5a385cb03e8aaf0b41ee595c410193686928f)
<img width="1721" alt="Screen Shot 2023-09-12 at 20 22 27" src="https://github.com/elastic/kibana/assets/43350163/136d7b7e-198c-44be-b521-d5e53da4ddfe">